SpringEx/SpringExAutowiringByType/bin/beans.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+
+<beans xmlns = "http://www.springframework.org/schema/beans"
+   xmlns:xsi = "http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation = "http://www.springframework.org/schema/beans
+   http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
+
+   <!-- Definition for textEditor bean -->
+   <bean id = "textEditor" class = "com.metacube.training.TextEditor" autowire = "byType">
+   <property name="spellChecker" ref= "spellCheckerId" />
```

Comment:
Refer https://www.tutorialspoint.com/spring/spring_autowiring_bytype.htm to check Autowiring byType